src/functions/databaseAccess.ts

```diff
@@ -40,31 +40,45 @@ export async function userHasPortfolios(userId: string): Promise<boolean>{
 
 export async function getAllPortfolios(): Promise<{photo: string, fullname: string, role: string}[]> {
     try {
-      const snapshot = await getDocs(
-        query(collection(db, PORTFOLIOS_COLLECTION_NAME), orderBy('createdTimestamp', 'desc'), where("status", "==", PortfolioStatus.PUBLISHED))
-      );
-      const docs: {photo: string, fullname: string, role: string}[] = [];
-      for (const doc of snapshot.docs) {
-        const docData = doc.data();
-        console.log(docData);
-        const photoUrl = await getImageUrlByPath(docData.photoPath);
-        if (!photoUrl) {
-          throw new Error("Could not get image link for path: " + docData.photo);
-        }
-        docs.push({
-          photo: photoUrl,
-          fullname: docData.fullName,
-          role: docData.role,
-        });
-      }
-      return docs;
-
+        return await getPortfolios([]);
     } catch (error) {
       console.error('Error fetching portfolios: ', error);
       return [];
     }
   }
 
+  export async function getPortfolios(constraints: QueryFieldFilterConstraint[]): Promise<{photo: string, fullname: string, role: string}[]> {
```

Comment:
For university project it doesn't matter, but if you create helper function just for internal usage in the module, don't export it. Or if it is not the helper function and it is assumed that it will be used in components, don't use third-party interfaces (QueryFieldFilterConstraints argument).